azavea/cac-tripplanner-android
# Feature/kak/search bar#19

## Overview

Adds a search widget to several activities, along with a content provider for searching GoPhillyGo places and events by name. Clicking a suggested place or event goes to its detail page.

## Demo

![gpg_search](https://user-images.githubusercontent.com/960264/42184569-fef1d98c-7e13-11e8-8cec-c811db9e52fd.gif)

## Testing

 - Click on magnifying glass anywhere it appears in the app toolbar
 - Type in part of a place or event name to search for
 - Click on one of the results
 - Should be taken to the detail page for the selection

Closes #19
